code/game/machinery/vending_types.dm

```diff
@@ -505,12 +512,12 @@
 		/obj/item/weapon/material/knife = 3,
 		/obj/item/weapon/reagent_containers/food/drinks/drinkingglass = 8,
 		/obj/item/clothing/suit/chef/classic = 2,
+		/obj/item/weapon/material/kitchen/rollingpin = 2,
```

Comment:
I'm not sure why a rolling pin was in the contraband section in the first place. unless I missed the reasoning behind it.